### DISCUSSION TITLE
Cannot update identity column 'id' [Apr 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5452] [Discord Threads]
> 🙋‍♂️ **CrowZ**  
#### when I'm only trying to update only this field  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1081123462854234142/image.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **kyleecodes**  
#### - RequestError: update [nc_zxkd___RegisteredUser] set [id] = @p0, [FullName] = @p1 where [id] = @p2;select @@rowcount - Cannot update identity column 'id'    
#### - Data Source: MS SQL  
#  
> 🙋‍♂️ **kyleecodes**  
#### Is your version of NocoDB upgraded?  
#  
> 🙋‍♂️ **CrowZ**  
#### this is what i got, in my nocodb  
![image0](https://cdn.discordapp.com/attachments/1081123463089111040/1081382128434020362/image.png)  
#  
> 🙋‍♂️ **CrowZ**  
#### i'm using n8n,   
#### and use noco as its node  
#  
> 🙋‍♂️ **CrowZ**  
#### I only send Fullname as field needed to update,   
#### but the query still set Id and fullname  
#  
> 🙋‍♂️ **CrowZ**  
#### If i'm using the swagger i also get the same error,   
#### if I send the id + fullnaem  
![image0](https://cdn.discordapp.com/attachments/1081123463089111040/1081383057816297572/image.png)  
#  
> 🙋‍♂️ **CrowZ**  
#### And without the id,   
#### it works just fine, if im using the swagger api  
![image0](https://cdn.discordapp.com/attachments/1081123463089111040/1081383276654116986/image.png)  
#  
> 🙋‍♂️ **CrowZ**  
#### but if using, n8n and nocodb as its node,   
#### And I only set only **FullName**.   
####    
#### Somehow it always think that i want to set id column   
####    
#### Screenshot attached  
![image0](https://cdn.discordapp.com/attachments/1081123463089111040/1081384294741721200/image.png)  
#  
> 🙋‍♂️ **kyleecodes**  
#### Can you give us your Project Info? https://docs.nocodb.com/FAQs#how-to-check-my-project-info-    
####    
#### Tagging <@&682859083799199814>  
#  
> 🙋‍♂️ **CrowZ**  
#### Here it is   
#### Node: **v16.19.1**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **true**   
#### RootDB: **mssql**   
#### PackageVersion: **0.105.3**  
#  
> 🙋‍♂️ **CrowZ**  
####   
![image0](https://cdn.discordapp.com/attachments/1081123463089111040/1081735339074404414/image.png)  
#  
> 🙋‍♂️ **CrowZ**  
#### this is the value from the node  
![image0](https://cdn.discordapp.com/attachments/1081123463089111040/1081735710542942318/image.png)  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1081123463089111040)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
